src/main/java/com/joseeduardo/todoist/controller/TarefaController.java

```diff
@@ -0,0 +1,41 @@
+package com.joseeduardo.todoist.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.joseeduardo.todoist.model.CriarTarefaEntradaDTO;
+import com.joseeduardo.todoist.service.TarefaService;
+
+@RestController
+@RequestMapping(value = "/v1/tarefa")
+public class TarefaController {
+
+	@Autowired
+	private TarefaService tarefaService;
+
+	@Autowired
+	public TarefaController(TarefaService tarefaService) {
+		this.tarefaService = tarefaService;
+	}
+
+	@PostMapping("/{id}")
+	public ResponseEntity<Void> criar(@RequestBody CriarTarefaEntradaDTO criarTarefaEntradaDTO,
+			@PathVariable("id") Long id) {
+		tarefaService.criar(criarTarefaEntradaDTO, id);
+		return ResponseEntity.status(HttpStatus.CREATED).build();
+	}
+
+	@DeleteMapping("/{id}")
+	public ResponseEntity<Void> exlcuirPorID(@PathVariable("id") Long id) {
+		tarefaService.excluirPorID(id);
```

Comment:
Quando você tem um método que recebe um parâmetro, nem sempre é necessário colocar o nome desse parâmetro no nome do método.

Exemplo:
`estamparCamisa(camisa)`

Seria menos redundante dessa forma:
`estampar(camisa)`

da mesma forma, você tem seu método 
`tarefaService.excluirPorID(id);`

Poderia ser:
```suggestion
		tarefaService.excluir(id);
```

Já fica claro que é por ID ;)

Uma sugestão para alterar apenas uma vez e já alterar tudo: Alt + Shift + R e altere. Ouuu, botao direito no nome do método -> Refactor > Rename (talvez seja rename all references).